examples/stable_diffusion_xl/gm/modules/embedders/modules.py

```diff
@@ -65,6 +60,12 @@ def ucg_rate(self):
     def input_key(self):
         del self._input_key
 
+    def freeze(self):
```

Comment:
may change to `raise NotImplementError`

Reply:
Multiple embedders share the same weights freezing logic. Those embedders that use a different way for freezing can override the parent method.

Reply:
em.. from coding cocern it might be dangerouse if we add a new inherited class and this method can be easily forget to be checked.. And no many embedder use this structure like `self.model.xxx`